util/devel/chplspell-dictionary

```diff
@@ -8314,6 +8315,7 @@ opengroup
 openhigh
 openlow
 openmem
+openMemFile
```

Comment:
Heh, thanks for being so conscientious!  Though in this case it's not needed -- the scspell utility that chplspell uses tokenizes the words it's checking on lowercase-to-uppercase transitions, so it'll check this as `open` `mem` `file`.  "Open" and "file" are of course fine, and since "mem" is less than four characters it won't even be checked.